hs_collection_resource/tests/test_collection.py

```diff
@@ -366,25 +366,6 @@ def test_update_collection_edit_permission(self):
         self.assertEqual(self.resCollection.resources.count(), 1)
         self.assertIn(self.resGen3, self.resCollection.resources.all())
 
-    def test_adding_one_collection_to_another_collection(self):
```

Comment:
I would have kept this test as it tests the code as it is now in the current implementation. Should remove only when we implement code to allow collection to be added to another collection.